src/api/call-api.test.ts

```diff
@@ -246,7 +246,7 @@ describe('callApi', () => {
     const buildAndExecuteRequestSpy = jest.spyOn(adapter, 'buildAndExecuteRequest') as any;
     const apiResponse = { success: true, result: '723.392028' };
     buildAndExecuteRequestSpy.mockResolvedValueOnce({
-      data: apiResponse,
+      ...apiResponse,
```

Comment:
@acenolaza  Perhaps you could confirm if this `apiResponse` value structure change is fine (since [the `GoResult` and `GoResultSuccess`](https://github.com/api3dao/promise-utils/blob/main/src/index.ts#L4) formats are different now)

Reply:
I think this is only changing the structure of the log message [here](https://github.com/api3dao/airkeeper/pull/35/files#diff-3e81ddd5a282280de51c1ef85c75c1275ed854660d5f853f13f8527ce6f0ff9fL63) because I am stringifying `apiResponse.data` instead of the whole `apiResponse` object (which has the structure of `{ success: true, data: {...} }`).

```ts
const message = `Failed to extract or encode value from API response: ${JSON.stringify(apiResponse.data)}`;
```

So stringifying `apiResponse` in the message would be another option if we want to include the `data: {...}` wrapper.

Reply:
The actual `data` is now under `apiResponse.data.data` which is weird 🤔

Reply:
> I think this is only changing the structure of the log message [here](https://github.com/api3dao/airkeeper/pull/35/files#diff-3e81ddd5a282280de51c1ef85c75c1275ed854660d5f853f13f8527ce6f0ff9fR65) because I am stringifying `apiResponse.data` instead of the whole `apiResponse` object (which has the structure of `{ success: true, data: {...} }`).
> 
> ```ts
> const message = `Failed to extract or encode value from API response: ${JSON.stringify(apiResponse.data)}`;
> ```
> 
> So stringifying `apiResponse` in the message would be another option if we want to include the `data: {...}` wrapper.

I don't think we want to JSON.stringify the whole AxiosResponse object